instrumentation/apache-camel-2.20/javaagent/src/main/java/io/opentelemetry/javaagent/instrumentation/apachecamel/CamelPropagationUtil.java

```diff
@@ -58,8 +58,9 @@ static void injectParent(Context context, final Map<String, Object> exchangeHead
     }
 
     @Override
-    public String get(Map<String, Object> map, String s) {
-      return (map.containsKey(s) ? map.get(s).toString() : null);
+    public String get(Map<String, Object> map, String key) {
+      Object value = map.get(key);
+      return (value == null ? null : map.get(key).toString());
```

Comment:
You still have the double get!  :). You can just say `value.toString()` here.

Reply:
Too fast, too furious 🙈 But in reality I probably should stop working after 6 PM 😬